.github/CODEOWNERS

```diff
@@ -907,6 +907,9 @@
 # ServiceLabel: %WebPubSub
 # ServiceOwners:                                                   @vicancy @JialinXin
 
+# ServiceLabel: %HardwareSecurityModules
```

Comment:
@emmeliaAra:  Please move this down into the Management section below, preserving alphabetical ordering by tag.

You're also missing the PRLabel block.   Please follow the format of existing entries below.

For example:
```suggestion
# PRLabel: %Hardware Security Modules
/sdk/hardware/secirutymodules/Azure.ResourceManager.*/             @emmeliaAra 
# ServiceLabel: %Hardware Security Modules %Mgmt
# ServiceOwners:                                                   @emmeliaAra
```

Reply:
Updated service name and moved it to the 'H' section